config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
+  get 'likes/create'
+  get 'comment/new'
+  get 'comment/create'
   root to: 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:index, :create]
```

Comment:
- [ ] There should be a `new` route which is handled by the `new` method for the Comments. Please kindly include a `new` route for the Comments.

Reply:
To add a comment navigate to the single post ( click on the post title) then there is a form where you can create a comment, the view file for the comment form is in the shared partials, as I am rendering it as a partial to the `post/show.html.erb`  view. The like button is also in this single post view.